src/extension.ts

```diff
@@ -0,0 +1,40 @@
+'use strict';
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import {spawnSync} from 'child_process';
+
+export function activate(context: vscode.ExtensionContext) {
+  console.log('Activated!');
+  vscode.languages.registerDocumentFormattingEditProvider('serenity-gml', {
+    provideDocumentFormattingEdits(document: vscode.TextDocument) {
+      // FIXME: Find a way how we can avoid forcing a save
+      document.save();
+      let filename = document.fileName;
+      let root_path = vscode.workspace.rootPath;
```

Comment:
Although this is a Serenity extension, please use TS/JS conventions and use CamelCase.